src/Modules/CalcDefence.lua

```diff
@@ -81,10 +81,29 @@ function calcs.defence(env, actor)
 	output.DamageReductionMax = modDB:Override(nil, "DamageReductionMax") or data.misc.DamageReductionCap
 	output.PhysicalResist = m_min(output.DamageReductionMax, modDB:Sum("BASE", nil, "PhysicalDamageReduction"))
 	output.PhysicalResistWhenHit = m_min(output.DamageReductionMax, output.PhysicalResist + modDB:Sum("BASE", nil, "PhysicalDamageReductionWhenHit"))
+
+	-- Highest Maximum Elemental Resistance for Melding of the Flesh
+	local highestResistMax = 0;
+	local highestResistMaxType = "";
```

Comment:
These variables can be tucked inside the if statement since they're not needed outside of it